src/libs/ReportUtils.js

```diff
@@ -279,11 +279,12 @@ function isSettled(reportID) {
         return false;
     }
     const report = allReports[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`] || {};
-    if ((typeof report === 'object' && Object.keys(report).length === 0) || report.isWaitingOnBankAccount) {
+    if (typeof report === 'object' && Object.keys(report).length === 0) {
```

Comment:
Why ` || report.isWaitingOnBankAccount` is removed here? This makes all pending requests as settled

Before:
<img width="1177" alt="Screenshot 2023-10-23 at 2 35 55 PM" src="https://github.com/Expensify/App/assets/108292595/9aab1af2-58ea-4a8f-bc98-f9593cf7c4c6">



After:
<img width="1177" alt="Screenshot 2023-10-23 at 2 36 29 PM" src="https://github.com/Expensify/App/assets/108292595/6f14d6b2-fd26-48ff-9e94-e3c39e9cf30b">

Reply:
Yeah, we just resolved this in DM, I pointed that out here: https://github.com/Expensify/App/pull/30090/files#r1368314158